Gemfile.lock

```diff
@@ -238,8 +256,11 @@ DEPENDENCIES
   capybara (>= 2.15)
   devise
   dotenv-rails
+  faker!
   jbuilder (~> 2.7)
   listen (>= 3.0.5, < 3.2)
+  minitest-rails (~> 6.0.0)
```

Comment:
What does this do? Just curious

Reply:
Test lib, was outdated and wouldn't pass until fixed